core/src/main/scala/org/apache/spark/scheduler/DAGScheduler.scala

```diff
@@ -716,11 +735,14 @@ class DAGScheduler(
     }
   }
 
-  /** Finds the earliest-created active job that needs the stage */
-  // TODO: Probably should actually find among the active jobs that need this
-  // stage the one with the highest priority (highest-priority pool, earliest created).
-  // That should take care of at least part of the priority inversion problem with
-  // cross-job dependencies.
+  /**
+   * Finds the earliest-created active job that needs the stage.
+   *
+   * TODO: Probably should actually find among the active jobs that need this
```

Comment:
I'm not sure all of this should be scaladoc. This looks like a private comment in the source code. Same for "Broken out for easier testing..."